cmd/lotus-storage-miner/sectors.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 var storeGarbageCmd = &cli.Command{
```

Comment:
there are still a bunch of things called store garbage internally. We will want to change those too

Reply:
Done.